frontend/test/test.helpers.ts

```diff
@@ -59,7 +59,7 @@ export const createTestStore = () => {
 		return waitingActions.length === 0;
 	};
 
-	const mainReducer = combineReducers(reducers);
+	const mainReducer = createReducer({});
```

Comment:
Why was this change necessary?

Reply:
All the v4 redux data was missing in the testing environment. This was to fix that